### PR TITLE
Create "Change account info" series

### PR DESCRIPTION
**Pull request by DonNateR**

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-2fdea54b4a/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-2fdea54b4a/how-to/)

File: content/general/change-account-company-name.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+**Note:** If you are a Dedicated Hosting customer, you can request all of the forms described in this article from your account manager.
+
+**Note:** Before completing the service transfer form, read the "Best practices for legal change forms" section of [Change account information](/how-to/change-account-information).
+
+1.  In the upper-right corner of the Cloud Control Panel, click **Account** > **Docs and Forms**.
+2.  Select the **Rackspace Account Change Forms** section.
+3.  In the **Transfer Ownership to a new Account owner** section, select one of the following options:
+
+  - If you are a Fanatical Support for AWS customer, download the **Fanatical Support for AWS Service Transfer Form**.
```
Comment (stephamon) [inline content/general/change-account-company-name.md]: These need to be indented to line up with "In the..." in step 3. As it is, they're causing the list to reset at 1 after the bullets.

File: content/general/transfer-account-ownership-or-tax-id.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+**Note:** If you are a Dedicated Hosting customer, you can request all of the forms described in this article from your account manager.
+
+**Note:** Before completing the service transfer form, read the "Best practices for legal change forms" section of [Change account information](/how-to/change-account-information).
+
+1.  In the upper-right corner of the Cloud Control Panel, click **Account** > **Docs and Forms**.
+2.  Select the **Rackspace Account Change Forms** section.
+3.  In the **Transfer Ownership to a new Account owner** section, select one of the following options:
+
+  - If you are a Fanatical Support for AWS customer, download the **Fanatical Support for AWS Service Transfer Form**.
```
Comment (stephamon) [inline content/general/transfer-account-ownership-or-tax-id.md]: These need to be indented so the bullet dash lines up with "In the..." in step 3.

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-3b8f8694af/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-07815c19c5/how-to/)

**Comment by stephamon** [comment]
👍

**Pull request merged**